content/r/concepts/csv-files/terms/colnames/colnames.md

```diff
@@ -0,0 +1,78 @@
+---
+Title: 'colnames()'
+Description: 'Returns or sets the names of the columns in a matrix-like object.'
```

Comment:
This function is not unique to CSV files, so I don't think it should be a term under CSV. I think we need a entry on data frames or matrix-like objects that this can go under.